src/db/migrations/schema.sql

```diff
@@ -51,6 +51,5 @@ CREATE TABLE users_messages_emojis (
     messages_id int NOT NULL,
     emoji varchar(100) NOT NULL, 
     FOREIGN KEY(users_id) REFERENCES users(id) ON DELETE CASCADE,
-    FOREIGN KEY(messages_id) REFERENCES messages(id) ON DELETE CASCADE,
-    PRIMARY KEY(users_id, messages_id, emoji)
```

Comment:
i had to remove th primary key because it wouldn't let me add the same emojis into the database if i had already reacted with one.

Reply:
hm... so the primary key is a composite and it just ensures that each entry is unique based on the combination of the user id, message id and emoji. there isn't a scenario where we'd want to allow a user to react to the same message with the same emoji more than once since it'll be nice to keep a count of each emoji reaction like slack, so that's why the primary key is defined as such. if you have a different user and they react with the same emoji to that same message, that isn't an issue since it doesn't violate the primary key constraint and is a unique reaction to the message. does that make sense?

Reply:
i'd keep the primary key constraint. let me know if that was unclear or if you have questions though.

Reply:
oh yeah, nevermind. i thought that and thought i had tested it yesterday with the same emoji on a different chat but i guess not?

Reply:
yeah, it had to have been with the same user, message and reaction, haha, since you can have the same user react to different messages in the same (or a different) chat with the same emoji reaction. but it's alright - ty for commented on it directly in the pr(!) so i wouldn't miss it.